### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by sercansimsek**
- [DEMO LINK](https://sercansimsek.github.io/layout_calendar/)
- [TEST REPORT LINK](https://sercansimsek.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

**Review by IoghiStudio — APPROVED** [review]
Look good!

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,42 @@
...
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
+        <div class="calendar__day"></div>
```
Comment (meforoma) [inline src/index.html:43]: @sercansimsek you seem to be missing few days ))
`<div class="calendar calendar--start-day--sun calendar--month-length--31">`
would not show 31 day )))

**Review by meforoma — CHANGES_REQUESTED** [review]
you've done great, but you're missing few days =)

let's add and be done

File: src/index.html (hunk 1 above)
Comment (sercansimsek) [reply src/index.html:43]: Gonna work on it

**Pull request closed without merging**